xml/dm_registers.xml

```diff
@@ -241,7 +241,10 @@
             to the module may fail. Specifically, \FdmDmstatusVersion may not return
             correct data.
 
-            1: The module functions normally.
+            1: The module functions normally. After writing 1, the debugger should
+            read \RdmDmcontrol to confirm that this bit is now high. Hardware may
+            take a long time to set this bit if it's not currently ready for
+            debugging to take place.
```

Comment:
This isn't really crucial but I find this sentence to be a little less formal than specs normally are (like "long time" and what does it mean for "debugging to take place").  How about something like:
"Hardware may postpone setting this bit until it is in a state that allows further debugger accesses."
or combining with a rewrite of the previous sentence:
"After this bit is written with 1, the bit will continue to be seen by reads as being 0 until the hardware is in the defined post-reset state.  Therefore, after writing 1 the debugger should poll \RdmDmcontrol until it observes that it is set."
(because the debugger shouldn't just read it once, it should poll until set)

Reply:
Or "After writing 1, the debugger should poll Dmcontrol until dmactive is high.  Hardware may take an arbitrarily long time to initialize and will indicate completion by setting dmactive to 1."

Reply:
Perfect.